src/android/notification/Options.java

```diff
@@ -57,6 +59,24 @@
  * methods to convert independent values into platform specific values.
  */
 public final class Options {
+    // Default Channel ID for SDK < 26
+    static final String DEFAULT_CHANNEL_ID = "channel-id-default";
+
+    // Silent channel
+    static final String CHANNEL_ID_SILENT = "channel-id-silent";
+    static final CharSequence CHANNEL_NAME_SILENT = "Silent Notifications";
+
+    // Vibrate only channel
+    static final String CHANNEL_ID_VIBRATE = "vibrate-channel-id";
```

Comment:
recommend to make the channel ID consistent. Maybe following the existing pattern?

In the current release, the CHANNEL_ID is "default-channel-id". We should make DEFAULT_CHANNEL_ID be that? Or is there a reason to change the channel name?

Then should use similar names e.g. "channel-id-vibrate", "channel-id-sound", "channel-id-sound-vibrate"

Reply:
I've addressed this in this PR, which contains some other features I need, as well as Android 10 device/SDK support:

https://github.com/katzer/cordova-plugin-local-notifications/pull/1891